plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListenerConfig.java

```diff
@@ -99,10 +103,37 @@ public Optional<String> getNamespace()
     }
 
     @Config("openlineage-event-listener.namespace")
-    @ConfigDescription("Override default namespace for job facet.")
+    @ConfigDescription("Override default namespace for job facet")
     public OpenLineageListenerConfig setNamespace(String namespace)
     {
         this.namespace = Optional.ofNullable(namespace);
         return this;
     }
+
+    public String getJobNameFormat()
+    {
+        return jobNameFormat.orElse(DEFAULT_JOB_NAME_FORMAT);
+    }
+
+    @Config("openlineage-event-listener.job.name-format")
+    @ConfigDescription("Set format for job name facet")
+    public OpenLineageListenerConfig setJobNameFormat(String jobNameFormat)
+    {
+        this.jobNameFormat = Optional.ofNullable(jobNameFormat);
+        if (jobNameFormat == null) {
+            return this;
+        }
```

Comment:
Airlift would invoke pass only non nullable String.

Reply:
But class method could be called with null passed directly